editor/src/components/inspector/editor-contract-section.spec.browser2.tsx

```diff
@@ -1,14 +1,14 @@
 import * as EP from '../../core/shared/element-path'
 import { assertNever } from '../../core/shared/utils'
-import { selectComponentsForTest } from '../../utils/utils.test-utils'
+import { selectComponentsForTest, wait } from '../../utils/utils.test-utils'
 import type { EditorContract } from '../canvas/canvas-strategies/strategies/contracts/contract-helpers'
 import { mouseClickAtPoint } from '../canvas/event-helpers.test-utils'
 import type { EditorRenderResult } from '../canvas/ui-jsx.test-utils'
 import { getPrintedUiJsCode, renderTestEditorWithCode } from '../canvas/ui-jsx.test-utils'
 import { groupSectionOption } from './editor-contract-section'
 
 const projectWithSizedDiv = `import * as React from 'react'
-import { Storyboard } from 'utopia-api'
+import { Storyboard, Group } from 'utopia-api'
```

Comment:
I understand why you have added `Group` to the imports here, but please can we have a test the ensures the import is automatically added

Reply:
sure!

Reply:
done in 54eef1672